authors/apps/articles/models.py

```diff
@@ -0,0 +1,19 @@
+from django.db import models
+from authors.apps.authentication.models import User
+
+from cloudinary.models import CloudinaryField
+
+class Article(models.Model):
+    """
+    This is the Article model that is used to handle CRUD on articles
+    """
+    title = models.CharField(max_length=200, blank=False)
+    body = models.TextField(blank=False, unique=True)
```

Comment:
Why unique=True?

Reply:
I wanted each article to be unique.

Reply:
Doesn't have to be. I don't see any need